numpy/f2py/setup.py

```diff
@@ -37,7 +37,7 @@ def _get_f2py_shebang():
     should be ``#!python`` rather than ``#!`` followed by the contents of
     ``sys.executable``.
     """
-    if set(('bdist_wheel', 'bdist_egg', 'bdist_mpkg', 'bdist_wininst',
+    if set(('bdist_wheel', 'bdist_egg', 'bdist_wininst',
             'bdist_rpm')).intersection(sys.argv):
         return '#!python'
```

Comment:
For `bdist_mpkg` we really do want `sys.executable` I think, because the thing being built will be unconditionally installed at a particular file-system location.

For the `bdist_wheel` etc - I think we need `#!python` for the reasons I just wrote up in the old PR https://github.com/numpy/numpy/issues/5834#issuecomment-99184620